test/cypress/integration/03-expenses.page.test.js

```diff
@@ -1,3 +1,155 @@
+import { randomSlug } from '../support/faker';
+
+describe('New expsense flow ', () => {
+  const collectiveSlug = randomSlug();
+
+  before(() => {
+    // Inital setup Creating a mock collective
+    cy.login({ redirect: '/brusselstogetherasbl' });
+    cy.get('[data-cy="host-apply-btn"]:visible').click();
+    cy.get(`input[name="name"]`).type('new-collective');
+    cy.get(`input[name="slug"]`).type(collectiveSlug);
+    cy.get(`input[name="description"]`).type('short description for new collective');
+    cy.get('[data-cy="custom-checkbox"]').click();
+    cy.wait(300);
+    cy.get('button[type="submit"]').click();
+    cy.wait(1000);
```

Comment:
This one can be achieved in a more performant way with `cy.createHostedCollective`

Reply:
Again the same issue here, Host would be opencollective and then I have to apply to brussles and then accept it using custom script , I guess its better this way

Let me know if I still need to change it